pkg/materialize/source.go

```diff
@@ -23,6 +24,44 @@ func GetTableStruct(v []interface{}) []TableStruct {
 	return tables
 }
 
+func DiffTableStructs(arr1, arr2 []interface{}) []TableStruct {
+	var difference []TableStruct
+
+	for _, item1 := range arr1 {
+		found := false
```

Comment:
`found` is redundant here because the only case  where you enter the second block is when you haven't encountered a `break`

Reply:
I tried removing it but getting different results. What were you thinking in how it should be written?